### PR TITLE
Group 3 Merge

### PR DESCRIPTION
**Pull request by Jacob-Allebach**
## Description

Define the overall conduct, decisions, and postmortems sections as well as the acceptable and unacceptable conduct for each.

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @Jacob-Allebach 
- @Ochirsaikhan 
- @anthonygc 
- @DlHolland

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Review by PaigeCD — DISMISSED** [review]
Looks good to me! Thanks for all your hard work, guys! ✨

File: conduct.md [hunk 1]
```diff
@@ -4,19 +4,21 @@
...
+- Students should finish work on time and ask for help if it is needed.
+- Students should be attentive in class and utilize the material to finish assignments.
+- Students should come to class having done the readings and be prepared to engage in discussions or activities.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not hold onto their opinions to avoid hurting someone's feelings.
```
Comment (mariakimheinert) [inline conduct.md]: Could someone from Team 3 clarify what this means? It reads as if students should say whatever they think regardless of whether it hurts someone's feelings or not, but I may be reading it wrong.

**Review by pkelly55 — DISMISSED** [review]

File: conduct.md (hunk 1 above)
Comment (Jacob-Allebach) [reply conduct.md]: That seems to be how the wording would imply it. Line 20 is also just a better phrased version of this, so we should probably get rid of this line if that works best.

File: conduct.md [hunk 2]
```diff
@@ -84,35 +86,37 @@ TODO: Write an introductory paragraph.
...
+- Students should not take final decisions personally.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not talk over others or prevent others from submitting their ideas.
+- Students should not commit and push code that falls outside of the group's initial intent for a project.
+- Students should update documentation or code in ways that are consistent with the decisions made by the team.
```
Comment (laurennevill) [inline conduct.md]: This last point doesn't follow the "should not" format like the rest of the unacceptable conducts.

Comment (laurennevill) [reply conduct.md]: An easy fix would be to write something like "Students should not update documentation or code in ways that are inconsistent with the decisions made by the team". Other than that, this looks good!

**Review by pkelly55 — APPROVED** [review]

**Review by lukel43 — APPROVED** [review]
Looks good to me!

**Review by PaigeCD — APPROVED** [review]

**Review by laurennevill — APPROVED** [review]

**Review by mariakimheinert — APPROVED** [review]
These additions to the conduct guide look great! Thanks for all your hard work, Team 3! :shipit:

**Review by noorbuchi — APPROVED** [review]

**Pull request merged**